html-css/week1/Nina/portfolio/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Nina's Portfolio Website</title>
+  <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+
+  <nav class="home">
+    <ul id="menu">
+      <li>
+        <a href="#home">Home</a>
+      </li>
+      <li>
+        <a href="#skills">Skills</a>
+      </li>
+      <li>
+        <a href="#projects"> Projects </a>
+      </li>
+      <li>
+        <a href="#contact" class="button contact-me-button">Contact Me</a>
+      </li>
+    </ul>
+  </nav>
+
+  <main>
+    <section class="hero container">
+        <div class="hero-pink">
+          <img src="./imgs/hero-img.png" alt="Nina Lin" width="100%" />
```

Comment:
```suggestion
          <img src="./imgs/hero-img.png" alt="Nina Lin" />
```
The image default width is always 100% as a block element.

Very good practice with adding `alt` attribute.